core/plugin_api.rs

```diff
@@ -10,6 +10,7 @@
 
 pub use crate::Op;
 pub use crate::OpId;
+pub use crate::OpResponse;
```

Comment:
Seems like unintended change

Reply:
Yeah, I was going to start working on making this work for plugins and then opted to leave it for a follow-up PR

Reply:
Actually that's only part of the story, I re-exported `OpResponse` here because in `test_plugin` and supposedly all plugins, their imports come from `deno_core::plugin_api::*`.

So the convention appears to be: re-exporting the subset of core types used by plugins here.